### DISCUSSION TITLE
Higher resolution or Lower resolution inference

### DISCUSSION THREAD

**Discussion by Aoi-00** [ultralytics/yolov5#8382] [Q&A]
Hi! I am currently using yolov5 for my final year project, so I am hoping to get some guidance on it. I am using RPI to capture a video stream of a pre-recorded footage being played on a separate screen (ipad/laptop), and streaming it to my PC for yolov5 recognition. Currently, I set the video stream to be 1920x1080 as I thought it would help the yolov5 get better accuracy with higher resolution images, as the captured images are a bit grainy.

However, I read that the default pretrained yolov5s model is trained on 640x480 images so I am wondering if I should have just set the stream to 640x480 for better accuracy? Did setting a higher resolution of 1920x1080 actually made the yolov5 accuracy worse? 

Additionally, right now I am only making use of the chair, human, and bottle/bag classes of the cocomo model. If the accuracy does not turn out well for the bottle/bag detection, is there a way to keep the pretrained model for chair & human, while adding a custom trained bottle/bag model and use these 2 on the images? Or would I have to retrain the entire model using a custom dataset for these 4 classes (I'd like to keep the human and chair model as they are working really well now! Thank you for making such an adaptive model!)?

Thank you so much for taking your time to read this and apologies for taking up your time!

**Comment by glenn-jocher** [comment]
@Aoi-00 train and deploy at similar --imgsz for best results
